00011_multiplication_table_12.js

```diff
@@ -0,0 +1,14 @@
+// 11. Write a program that prints a multiplication table for numbers up to 12.
+
+const multiplicationTable = () => {
+    for (let i = 1; i < 13; i++) {
+        let tableOf = i;
```

Comment:
Why you are declaring the variable here?

Reply:
Is there any problem to declare variables locally?. I only use that variable inside of this function. Is that an issue?